src/pymor/discretizers/builtin/cg.py

```diff
@@ -995,6 +995,11 @@ def discretize_stationary_cg(analytical_problem, diameter=None, domain_discretiz
         Must be provided if `grid` is specified.
     preassemble
         If `True`, preassemble all operators in the resulting |Model|.
+    mu_energy_product
+        If not `None`, |parameter values| for which to assemble the symmetric part of the |Operator| of the resulting
+        |Model| `fom` (ignoring the advection part), such that `fom.operator.assemble(mu) == fom.products['energy']`
```

Comment:
This sounds a bit strange. First you say, that the advection part is ignored, then you say, that both operators are the same ...